src/pages/uxp-api/reference-js/Global Members/HTML Events/EventTarget.md

```diff
@@ -7,25 +7,27 @@
 
 <a name="eventtarget-addeventlistener" id="eventtarget-addeventlistener"></a>
 
-## addEventListener(eventName, callback, [capture])
+## addEventListener(eventName, callback, options)
+**See**: [EventTarget - addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener)  
 
-| Param | Type | Default |
+| Param | Type | Description |
 | --- | --- | --- |
-| eventName | `*` |  | 
-| callback | `*` |  | 
-| [capture] | `boolean` | `false` | 
+| eventName | `*` |  |
+| callback | `*` |  |
+| options | `boolean` \| `Object` | Boolean value denoting capture value or options object. Currently supports only capture in options object ({ capture: bool_value }). |
```

Comment:
@bhatiaPankaj  @kareemmyk  Can you confirm if these changes are intentional? The change to eventTarget also impacts a lot of HTMLElements.